### PR TITLE
add e-mail notification service and enforce code formatting rules

### LINKED ISSUE #1: Add e-mail notification service
Implement an e-mail notification service, i.e. MailGun

Use case:
When any monitor yields a valid result, trigger send a e-mail notification containing the monitor description, the content returned by the monitor and timestamp.

Notification should be triggered from here: https://github.com/ofaltins/website-monitor/blob/master/src/runMonitor.js#L53

### LINKED ISSUE #6: Refactor: Enforce Code Formatting Rules
@ofaltins I have also seen that there is currently inconsistency in the code files. For example, `cli.js` has 2 space indentation while others have 4 spaces.

So I'm proposing a style rule in `.editorconfig`.

image

image

### PR DESCRIPTION
**Pull request by chankruze**
- [x] fixes #1
- [x] fixes #6

Signed-off-by: chankruze <chankruze@gmail.com>

### CODE REVIEW

**Comment by chankruze** [comment]
@ofaltins please review

File: src/utils/sendEmail.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+const sendEmail = (subject, body) =>
+  mailgunClient.messages
+    .create(DOMAIN, {
+      from: `${MAILGUN_EMAIL_FROM_NAME} <events@${DOMAIN}>`,
+      to: MAILGUN_EMAIL_FROM_TO,
+      subject,
+      text: body,
+    })
+    .then((res) => {
+      console.log(`Email sent to ${data.to}`);
```
Comment (ofaltins) [inline src/utils/sendEmail.js]: `data` is not defined here

File: src/utils/sendEmail.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      //   console.log(res);
+    })
+    .catch((err) => {
+      console.error(`Failed to send email to ${data.to}`);
```
Comment (ofaltins) [inline src/utils/sendEmail.js]: `data` is not defined here

File: src/utils/sendEmail.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+if (!process.env.MAILGUN_EMAIL_TO) {
+  throw new Error(
+    "Please set `MAILGUN_EMAIL_FROM` value in environment variable"
+  );
+}
+
+const DOMAIN = process.env.DOMAIN;
+const MAILGUN_EMAIL_FROM_NAME = process.env.MAILGUN_EMAIL_FROM_NAME;
+const MAILGUN_EMAIL_FROM_TO = process.env.MAILGUN_EMAIL_FROM_TO;
```
Comment (ofaltins) [inline src/utils/sendEmail.js]: `MAILGUN_EMAIL_FROM_TO` is not specified as an .env variable. Did you mean `MAILGUN_EMAIL_TO` ?

**Review by ofaltins — CHANGES_REQUESTED** [review]
Good job @chankruze - there are some minor things that needs fixing before merge - please have a look

File: src/utils/sendEmail.js (hunk 3 above)
Comment (chankruze) [reply src/utils/sendEmail.js]: Ya, I figured out that, I forgot to ammend the commit locally. Fixing it now.

**Pull request merged**